### ISSUE TITLE
Couldn't find an entry that matches the window title

### ISSUE THREAD

**Issue by iipamonkey** [keepassxreboot/keepassxc#4789]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )
After upgrade to 2.5.4 and performing auto-type, dialog says "Couldn't find an entry that matches the window title" for all entries and all browsers. This never happened for me prior to 2.5.4.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. open browser to required page
2. select database entry and perform auto-type
3. dialog below appears instead of successful auto-type

<img width="532" alt="Screen Shot 2020-05-28 at 10 45 10 PM" src="https://user-images.githubusercontent.com/20322778/83219367-cf364380-a135-11ea-90b8-688ee0f2e130.png">

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.5.4
Revision: dcca5aa

Qt 5.14.1
Debugging mode is disabled.

Operating system: macOS Mojave (10.14)
CPU architecture: x86_64
Kernel: darwin 18.7.0

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- TouchID

Cryptographic libraries:
 libgcrypt 1.8.5

**Comment by droidmonkey** [comment]
You need to invoke global auto-type from the window you want to auto-type into. So in this case, have the browser selected then hit your global auto-type hotkey. If you want to type a specific entry, go to keepassxc then press Cmd+Alt+V.

**Comment by iipamonkey** [comment]
> You need to invoke global auto-type from the window you want to auto-type into. So in this case, have the browser selected then hit your global auto-type hotkey. If you want to type a specific entry, go to keepassxc then press Cmd+Alt+V.

Thank you very much for explaining these last couple issues I had. Very much appreciated. I did look at the changelog and was surprised that nothing in there seemed to reference these items. Anyway, thanks a million for the awesome work on the best x-platform password manager!

**Comment by droidmonkey** [comment]
This has been how it has worked since the beginning.

**Comment by iipamonkey** [comment]
> This has been how it has worked since the beginning.

Now I'm thoroughly intrigued. I wish I could tell how long I've been using it; I remember I'd been using it a while when I got an update and was delighted to see macOS dark mode support, if that helps? Anyway, the entire time I've been using it (up until 2.5.4), I simply went to any entry in my database, invoked my auto-type shortcut with that entry highlighted, and it would alt-tab to any window I had just used and populate my user/password into whatever field I had placed the cursor in said window.

In all that time (almost a year'ish?), I have never invoked auto-type from the browser. I am undoubtedly crazy, but not so crazy as to not remember how I've used this tool 10-20X per day for a year! :P

In any case, I'm getting used to the new (or old, maybe? I don't know what to think lol) method.

Thanks again for all the work!
